cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -466,6 +465,31 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	repoTmpl.Execute(w, data)
 }
 
+func (s *Server) enqueueForIndex(queue *Queue)  func (rw http.ResponseWriter, r *http.Request) {
```

Comment:
a comment explaining why this exists (ie repo-updater/frontend will push new repos)